modules/SystemsBiology.py

```diff
@@ -0,0 +1,164 @@
+from settings import Config
+from common.HGNCParser import GeneParser
+import sys
+import logging
+import datetime
+import opentargets.model.core as opentargets
+import opentargets.model.bioentity as bioentity
+import opentargets.model.evidence.core as evidence_core
+import opentargets.model.evidence.association_score as association_score
+
+__copyright__ = "Copyright 2014-2018, Open Targets"
+__credits__   = ["Michaela Spitzer"]
+__license__   = "Apache 2.0"
+__version__   = "1.2.8"
+__maintainer__= "Michaela Spitzer"
+__email__     = ["data@opentargets.org"]
+__status__    = "Production"
+
+SYSBIO_DATABASE_ID='SYSBIO'
+SYSBIO_VERSION='2018.11'
+
+class SYSBIO:
+    def __init__(self):
+        self.evidence_strings = list()
+        self.symbols = {}
+        self.logger = logging.getLogger(__name__)
+
+    def process_sysbio(self):
+        gene_parser = GeneParser()
+        gene_parser._get_hgnc_data_from_json()
+
+        self.symbols = gene_parser.genes
+        self.build_evidence()
+        self.write_evidence()
+
+    def build_evidence(self, filename1=Config.SYSBIO_FILENAME1, filename2=Config.SYSBIO_FILENAME2):
+
+        now = datetime.datetime.now()
+
+        # *** Build evidence.provenance_type object ***
+        provenance_type = evidence_core.BaseProvenance_Type(
+            database=evidence_core.BaseDatabase(
+                id=SYSBIO_DATABASE_ID,
+                version=SYSBIO_VERSION
+            )
+        )
+        error = provenance_type.validate(logging)
+
+        if error > 0:
+            self.logger.error(provenance_type.to_JSON(indentation=4))
+            sys.exit(1)
+
+        # Build dictionary with publication info (pmid, method description, score type)
+        # TODO: Change to (pmid, gene set name, method description, score type) so that scores can be categorised per gene set
+        SYSBIO_PMID_MAP = {}
+        with open(filename2, 'r') as sysbio_publications:
+
+            for line in sysbio_publications:
+                # pmid	method	score_type
+                (sysbio_pmid, sysbio_method, score_type) = tuple(line.rstrip().split('\t'))
+                SYSBIO_PMID_MAP[sysbio_pmid]={'method':sysbio_method, 'score_type':score_type}
+
+        with open(filename1, 'r') as sysbio_input:
+            n = 0
+            for line in sysbio_input:
+                n +=1
+                if n>1:
+                    '''
```

Comment:
please, use a proper comment as `#` but not strings

Reply:
I know and whenever I see a comment that is a string I quickly change it to a proper comment. They have all been changed in the sysbio.py file.